app/models/organization.rb

```diff
@@ -9,28 +10,35 @@ def initialize(init_params)
     @name = init_params['name']
     @description = init_params['description']
     @imageUrl = init_params['profileImageUrl']
-    @categories = init_params['type']
-    @interests = init_params['categories']
+    @categories = init_params['categories']
+    @interests = init_params['interests']
   end
 
   def self.get_organizations(num, categories, interests)
     orgs = self.get_orgs_from_api()
-
+    csv_text = File.read('lib/interest.csv')
+    interest = CSV.parse(csv_text, headers: true)
     top_orgs = []
     orgs.each do |i|
       org_categories = i['categories']
       org_categories.each do |c|
         categories.each do |quiz_category|
           if c['categoryName'].downcase.include? quiz_category.downcase
+            org_interest = interest.find {|row| row['Organization ID'].to_i == i['organizationId']}
+            if org_interest != nil
+              i['interests'] = org_interest['Organization Interest']
+            else
+              i['interests'] = ''
+            end
             top_orgs.push(Organization.new(i))

```

Comment:
Looks like this being in a for loop is resulting in multiple of the same orgs being added as seen here 

<img width="1360" alt="Screen Shot 2019-05-11 at 5 45 47 PM" src="https://user-images.githubusercontent.com/19477044/57576469-2d9e6000-7415-11e9-97af-d2433713b316.png">

To reproduce :
Take Quiz:
Select Academic, Advocacy
Then Select first and last of each interest question